docs/components/protect.mdx

```diff
@@ -1,61 +1,65 @@
 ---
 title: <Protect />
-description: The <Protect /> component is used to only render content that a user with a role with the correct permissions can access.
+description: The <Protect /> component is used for authorization. It only renders its children when the current user has the specified permission or role in the organization.
 ---
 
 # `<Protect />`
 
-The `<Protect />` component only renders content that a user with [a Role with the correct Permissions](/docs/organizations/roles-permissions) can access.
+The `<Protect />` component is used for authorization. It only renders its children when the current user has the specified [permission](/docs/organizations/roles-permissions) or role in the organization.
 
 ## Usage
 
-You can pass either a `role` or `permission` to `<Protect />` to limit who can see the content it renders. If you do not pass either prop, `<Protect />` behaves the same as [`<SignedIn />`](/docs/components/control/signed-in) and will render its children if the user is signed in, regardless of their Role or its Permissions.
+You can pass either a `permission` or `role` key to `<Protect />` to limit who can see the content it renders. If you do not pass either prop, `<Protect />` behaves the same as [`<SignedIn />`](/docs/components/control/signed-in) and will render its children if the user is signed in, regardless of their Role or its Permissions.
 
 For more complex authorization logic, pass conditional logic to the `condition` prop.
 
-### Render content by Role
+### Render content by Permissions
 
-The children of the following component will only be visible to users with the `admin` Role.
+The children of the following component will only be visible to users with Roles that have the `org:posts:create` permission.
 
 <Tabs type="framework" items={["Next.js", "React"]}>
-  <Tab>
-    ```jsx
-    import { Protect } from "@clerk/nextjs";
-
-    export default function ProtectPage() {
-      return (
-        <Protect
-          role="org:admin"
-          fallback={<p>Only an admin can access this content.</p>}
-        >
-          {children}
-        </Protect>
-      );
-    }
-    ```
-  </Tab>
-
-  <Tab>
-    ```jsx
-    import { Protect } from "@clerk/react";
-
-    export default function ProtectPage() {
-      return (
-        <Protect
-          role="org:admin"
-          fallback={<p>Only an admin can access this content.</p>}
-        >
-          {children}
-        </Protect>
-      );
-    }
-    ```
-  </Tab>
+    <Tab>
```

Comment:
We use two spaces to indent all of our code. Please update this sample accordingly.